bot/cogs/core/paginator.py

```diff
@@ -1,34 +1,64 @@
-import asyncio
 import inspect
-from typing import Any, Callable, Coroutine
+from typing import Any, Callable, Coroutine, Optional, Union
 
 import discord
 from discord.ext import commands
 
 from bot.utils.ctx import Ctx
 from bot.villager_bot import VillagerBotCluster
 
-PAGE_EMBED_CALLABLE = Callable[[int], discord.Embed | Coroutine[Any, Any, discord.Embed]]
-LEFT_ARROW = "⬅️"
-RIGHT_ARROW = "➡️"
-SKIP_TO_LAST = "⏩"
-BACK_TO_FIRST = "⏪"
-NAV_EMOJIS = [BACK_TO_FIRST, LEFT_ARROW, RIGHT_ARROW, SKIP_TO_LAST]
+PAGE_EMBED_CALLABLE = Callable[[int], Union[discord.Embed, Coroutine[Any, Any, discord.Embed]]]
```

Comment:
It looks like you had some merge conflicts and then kinda rolled over some of the changes I made on Iapetus-11/main that your branch didn't have yet. 

For example, I replaced the `Union[...]` with the new syntax using a `|`, which got reverted. 

Could you change it back? Thanks!

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/281/commits/91325f729927197667f3f7a40b6f6727830f4b02